### PR TITLE
reciever revert

### PR DESCRIPTION
**Pull request by knooyala**
Changed version of reciever's index file according to https://jira.corp.ooyala.com/browse/PLAYER-3778. Added ability to set pcode && brandingId.

### CODE REVIEW

File: receiver.html [hunk 1]
```diff
@@ -47,6 +50,10 @@
                     }
                 )
         };
+        
+        // Your codes
```
Comment (aldigun) [inline receiver.html]: This is not the correct way to assign pcode and playerID params. These 2 must be inside var pageLevelParams = {}.
OO.__internal is internal use only, and not for customer use case.

File: receiver.html [hunk 2]
```diff
@@ -28,7 +31,7 @@
     <!-- Container for Player V4 -->
     <!-- Having this element on the page is required -->
     <div id="player"></div>
-    <script type="text/javascript" src="//player.ooyala.com/static/chromecast/cast.min.js"></script>
+    <script type="text/javascript" src="//player.ooyala.com/static/chromecast/staging/cast.min.js"></script>
```
Comment (aldigun) [inline receiver.html]: For github receiver.html and readme, let's use //player.ooyala.com/static/chromecast/cast.min.js

But staging appID D4C7BD4A and staging receiver.html indeed needs 
player.ooyala.com/static/chromecast/staging/cast.min.js

File: README.md [hunk 3]
```diff
@@ -80,12 +80,19 @@ var pageLevelParams = {
         )
 };
 
+// Your codes
```
Comment (aldigun) [inline README.md]: Remove these from readme, as per the comment on receiver.html

Comment (knooyala) [reply README.md]: +

**Comment by aldigun** [comment]
LGTM

**Pull request merged**